numba/tests/test_sort.py

```diff
@@ -923,5 +927,149 @@ def test_argsort_stable(self):
             self.check_argsort_stable(sorter, *args)
 
 
+nop_compiler = lambda x:x
+
+
+class TestSortSlashSortedWithKey(MemoryLeakMixin, TestCase):
+
+    def test_01(self):
+
+        a = [3, 1, 4, 1, 5, 9]
+
+        @njit
+        def external_key(z):
+            return 1. / z
+
+        @njit
+        def foo(x, key=None):
+            return sorted(x[:], key=key), x[:].sort(key=key)
```

Comment:
`.sort` is returning `None`. it's not testing the sorted result.

Reply:
fair point :)